ENGAGEHF/Dashboard/Vitals/Views/VitalsCard.swift

```diff
@@ -0,0 +1,109 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import HealthKit
+import SwiftUI
+
+
+struct VitalsCard: View {
+    @ScaledMetric private var quantityTextSize: CGFloat = 20
+    
+    let quantity: String?
+    let units: String?
+    let type: String?
+    let date: Date?
+    
+    private var displayDate: String? {
+        date?.formatted(date: .numeric, time: .omitted)
+    }
+    
+    
+    var body: some View {
+        StudyApplicationListCard {
+            VStack(alignment: .center, spacing: 10) {
+                Text(type ?? "--")
+                    .font(.headline)
+                    .foregroundColor(.primary)
+                    .multilineTextAlignment(.center)
+                HStack {
```

Comment:
I have worked a bit on testing out the UI to refine it and I think it might be redundant to display the type of information within the card. Weight, blood pressure, and heart rate might be clearly identifiable based on the value itself. I would propose to therefore remove the type here and just focus on the main measurement and text.
I oddly also suggest to make the two labels containing the value and unit closer to each other and first text baseline aligned to ensure that the unit's font is based line aligned with the text:
```suggestion
                HStack(alignment: .firstTextBaseline, spacing: 4) {
```